src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -28,7 +28,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
         // like this enables unit tests to assert different outcomes, based on whether it is set or not
         FEATURE_FLAG_ENABLE_REDIS_REMOVAL: true,
         activeSubmissionBasePath: getUrlWithParamsToPath(config.ACTIVE_SUBMISSION_BASE_PATH, req),
-        backLinkUrl: config.BENEFICIAL_OWNER_STATEMENTS_URL,
+        backLinkUrl: getUrlWithParamsToPath(config.BENEFICIAL_OWNER_STATEMENTS_WITH_PARAMS_URL, req),
```

Comment:
Lookslike it's missing a call to `isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL)`

Reply:
I think that call is on line 25.. That entire block is for code called when the `FEATURE_FLAG_ENABLE_REDIS_REMOVAL` is set to true.